electron/src/lib/CoreProtocol.ts

```diff
@@ -98,14 +100,18 @@ export class CustomProtocolHandler {
       });
     } else {
       app.once('ready', async () => {
+        logger.info('App ready, looking for deep link in arguments ...');
         const deepLink = this.findDeepLink(process.argv);
         if (deepLink) {
+          logger.info('App ready, dispatching deep link ...');
           await this.dispatchDeepLink(deepLink);
         }
       });
       app.on('second-instance', async (event, argv) => {
+        logger.info('Second instance detected, looking for deep link in arguments ...');
```

Comment:
```suggestion
        logger.info('Second instance detected, looking for deep link in arguments...');
```

Reply:
Not using this since we have spaces before ellipsis in the other log messages, too. Unless you have a good reason for that? :)